### ISSUE TITLE
CC Setup

### ISSUE THREAD

**Issue by XLuma** [Mirosta/CCPoker#1]
Is there a setup to how the computers/monitors should be placed for this to function properly ?

**Comment by Mirosta** [comment]
I've just added a readme file, hopefully it answers that, let me know if you have further questions.

**Comment by XLuma** [comment]
Are the pedestals actually required ? Or can be replaced by any pedestal ?

**Comment by XLuma** [comment]
Oh, and also the pokerRender throws out an error at line 148, since card_back_1 doesn'T exist. Haas that file been forgotten, or should we change the line to use the included card_back ?

**Comment by XLuma** [comment]
Yeah unfort there are a couple files missing (for example, arrow.nfp)

**Comment by XLuma** [comment]
Also, for an actual error: pokerClient cannot load player state, because there isn't a player.conf present(on the computer or pocket computer ?) and doesnt creates one at start. Not sure on the structure of this file too

**Comment by Mirosta** [comment]
Pedestals are eventually meant to be the mechanism to join the game. You would place a floppy disk containing your username and money onto the pedestal, a turtle will be given the disk, via a pipe, copy the config from the floppy disk to a personal computer, then put the personal computer into the pedestal.

The player can then pick up the computer and start it, which joins them to the game.

I have uploaded the missing files and an example player.conf

**Comment by XLuma** [comment]
Oooh, I see. very nice ! I'm playing on Enigmatica so I hope Reliquary get's updated for 1.16.4 soon.

**Comment by Mirosta** [comment]
You could just use chests as an alternative, although it wouldn't be so visually pleasing
